### PR TITLE
updated queries.js in frontend to fix typos and add euery single cate…

### PR DESCRIPTION
**Pull request by melinamboedecker**
…gory and label, updated mutation for ADD_USER with registerInput, and updated all comment body to commentText

### CODE REVIEW

**Review by Zoneam — APPROVED** [review]

File: client/src/utils/queries.js [hunk 1]
```diff
@@ -119,22 +119,32 @@ export const QUERY_ALL_CATEGORIES = gql`
...
 `;
 
-//For below query, do we need to put the posts in the Category typeDef?
-export const QUERY_CATEGORY_POSTS = gql`
-  query category(categoryId: ID!) {
-    category(categoryId: ID!) {
-      _id
-      name
-      posts {  
+//Can't include posts in single category search as posts aren't connected to the categories in typeDefs
```
Comment (vasylynash) [inline client/src/utils/queries.js:122]: Update typeDefs and check

File: client/src/utils/queries.js [hunk 2]
```diff
@@ -145,17 +155,28 @@ export const QUERY_ALL_LABELS = gql`
...
+//         body
+//         author
+//         dateCreated
+//       }
+//     }
+//   }
+// `;
+
+
+//Can't include posts in single category search as posts aren't connected to the categories in typeDefs
```
Comment (vasylynash) [inline client/src/utils/queries.js:174]: Update typeDefs and check

**Review by vasylynash — CHANGES_REQUESTED** [review]

File: client/src/utils/queries.js (hunk 1 above)
Comment (melinamboedecker) [reply client/src/utils/queries.js:122]: going to check this with enum on a different branch

File: client/src/utils/queries.js (hunk 2 above)
Comment (melinamboedecker) [reply client/src/utils/queries.js:174]: going to check this with enum on a different branch

**Pull request closed without merging**